Automattic/jetpack
# Boost: Reload on connect

Fixes 58-gh-Automattic/boost-cloud

#### Changes proposed in this Pull Request:

* Reload the window when jetpack connection is established for the site.

#### Other information:


#### Jetpack product discussion
p1655882229140969-slack-C016BBAFHHS

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Install the plugin on a fresh site.
* Go through the connection process.
* Make sure the page reloads on successful connection.
* Enable Critical CSS
* Go to the upgrade flow
* Make sure the upgrade flow doesn't skip the benefits page and does show the pricing.
